cLvgl/.vscode/launch.json

```diff
@@ -0,0 +1,162 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Local AMD64",
+            "type": "cppdbg",
+            "request": "launch",
+            "program": "${workspaceFolder}/build-local/bin/main",
+            "args": [],
+            "stopAtEntry": false,
+            "cwd": "${workspaceRoot}",
+            "environment": [],
+            "externalConsole": false,
+            "MIMode": "gdb",
+            "setupCommands": [
+                {
+                    "description": "Enable pretty-printing for gdb",
+                    "text": "-enable-pretty-printing",
+                    "ignoreFailures": true
+                },
+                {
+                    "description":  "Set Disassembly Flavor to Intel",
+                    "text": "-gdb-set disassembly-flavor intel",
+                    "ignoreFailures": true
+                }
+            ],
+            "preLaunchTask": "build-debug-local"
+        },
```

Comment:
I would drop this one, because as this is using the framebuffer would not be good to mess up with the local desktop /dev/fb0, or there is an easy way to run it local with some "virtual display"?